[IMP] events: full rewrite of essentials

ksc-odoo opened the pull request:
PROJECT TASK: https://www.odoo.com/web#id=3873094&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

This is going to be the first PR in a series of PRs that will constitute a complete re-write/refresh of the current Events documentation.

This first one will still use the 'event_essentials' RST file name, but will focus on the same in-depth material as the newly-written Events tutorial that covers Event Settings, Features, and Dashboards.
Then, that will be followed up by a specific PR for how to 'Create an Event', and so on...

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/8779).

ksc-odoo commented:
Hey, @larm-odoo -- this one is ready for a Peer Review from you whenever you get a chance! Thanks!

larm-odoo commented on content/applications/marketing/events/event_essentials.rst:
```diff
@@ -2,283 +2,384 @@
+exhibitors on event pages, which can serve as a valuable incentive to encourage partners and
+businesses to participate in the event.
+
+The :guilabel:`Jitsi Server Domain` field represents an external conferencing service that is
```
I have never heard of this before- not sure if there is other documentation (internal or external) to link to this? Just an idea.

ksc-odoo replied:
I actually wasn't familiar with it, either. Had to do some research just to figure it out. I also searched for any existing on doc about it, but came up empty. If anything gets created that mentions it, though, I'll be sure to toss it in 👍

larm-odoo reviewed (approved):
Great Doc @ksc-odoo! I just had some suggestions and a few questions, but I am sure you can  handle them. As always, if you have any questions on my comments, let me know. I am approving now!

ksc-odoo commented on content/applications/marketing/events/event_essentials.rst:
```diff
@@ -2,283 +2,384 @@
 
-Kanban concepts and organization
-================================
+Odoo *Events* is fully-integrated with other Odoo applications, which provides users with the
+ability to publish events on their website, sell tickets (online or via sales orders), and grant
+access to attendees via barcodes or QR codes. This all-inclusive application also allows users to
+send automated messages and social posts leading up to *and* after the event.
```
Great point! I plan on revisiting this portion to accomplish that once the other updated events docs get merged. Thanks for friendly reminder!

ksc-odoo commented on content/applications/marketing/events/event_essentials.rst:
```diff
@@ -2,283 +2,384 @@
+Kanban view
+-----------
+
+The Kanban view is the default view of the main :guilabel:`Events` dashboard, and is represented by
+the view option displayed as the :guilabel:`two downward-facing columns` icon, located in the
+upper-right corner.
```
I *fully* get where you're coming from. However, I think by describing the look of the icon/button, and where it is located on the page/UI, I think that should suffice in this instance. But i'll be sure to keep that kind of guidance in mind for future PRs, for sure 👍

ksc-odoo commented on content/applications/marketing/events/event_essentials.rst:
```diff
@@ -2,283 +2,384 @@
+   right of the other stages.
 
-When clicked, a drop-down menu of settings will appear.
+To add a new stage to the Kanban pipeline, click the :guilabel:`+ Stage` button, located to the
```
in this instance, since the plus sign on the UI isn't as pronounced as it is in other areas of the platform, I'm opting for the typed "+" sign here. One, that ensures that it shows up for every reader. Two, it more accurately represents that button, in my opinion. And three, it keeps that GUI clean, whereas otherwise, I'd have to add a (plus sign) mention within the GUI. 👍

ksc-odoo commented on content/applications/marketing/events/event_essentials.rst:
```diff
@@ -2,283 +2,384 @@
-.. image:: event_essentials/events-tickets-tab.png
+   :alt: The Stage text field when adding a new stage to the Odoo Events application.
+
+There are two ways to edit event Kanban stages.
```
Great catch! I fully forgot to add 'Configuration > Event Stages' and so on...Totally my fault. Just added that in. Thanks again for spotting this!

ksc-odoo commented on content/applications/marketing/events/event_essentials.rst:
```diff
@@ -2,283 +2,384 @@
-options, which are entered in the :guilabel:`Answers` tab below. The :guilabel:`Text Input` type
-allows attendees to write in their own answer to the question in a text box.
+The calendar view of the :guilabel:`Events` dashboard can be accessed by clicking the :guilabel:`📆
+(calendar)` icon in the upper-right corner. Doing so transforms the dashboard into a calendar
```
Again, I think the current explanation works, but if I get any additional feedback about this I will make that changes (AND give you total credit 😆)

ksc-odoo commented on content/applications/marketing/events/event_essentials.rst:
```diff
@@ -2,283 +2,384 @@
-Inside the invite template form, are options to sent email or SMS invites. Each message can be fully
-customized, and recipients can be added.
+If the user no longer wishes to add an event on this date, either click :guilabel:`Cancel` or
+:guilabel:`X` to remove the :guilabel:`New Event` pop-up window.
```
I think because I'd also have to use an (X) icon beside this an "emoji safety placeholder" -- I don't think the emoji is super necessary in this instance, but if I get the same note, I'll definitely make the change (and give you full credit again! 😆 )

ksc-odoo commented:
Thanks for your super-helpful feedback, @larm-odoo -- made all the necessary adjustments, and feel like this is now ready for Final Review @StraubCreative -- whenever you get a chance. Thanks! 🙏

ksc-odoo commented:
@StraubCreative thank you so much for the incredibly detailed feedback. Per your suggestion, I moved all the *original* content from the previous essentials doc to the TOC tree rst file, and made the necessary adjustments there. You'll notice that the original version's portions focused on creating an event are no longer there, because I've already created a new doc for that specific content, which has already been merged/published, so I only focused on the core elements of what can be found on the events dashboard -- making this a very succinct and to-the-point doc. All that to say, I've implemented the necessary adjustments, and believe this is now ready for another look from you whenever you get a chance. Thanks again for the always-helpful feedback. It is very much appreciated 🙏

StraubCreative commented on content/applications/marketing/events.rst:
```diff
@@ -7,9 +7,61 @@ Events
+   The :guilabel:`Ended` and :guilabel:`Cancelled` stages are folded, by default, and located to the
+   right of the other stages.
+
+On each event card, users can find the scheduled date of the event, the name of the event, the
```
suggestion: don't need "users" language throughout the doc
```suggestion
On each event card, find the scheduled date of the event, the name of the event, the
```

StraubCreative commented on content/applications/marketing/events.rst:
```diff
@@ -7,9 +7,61 @@ Events
+Each event card can be dragged-and-dropped into any stage on the Kanban pipeline, providing easy
+organizational access.
+
 .. toctree::
```
This will have to be fixed (or removed) as it's a bit of a mess [when rendered as html](http://runbot134.odoo.com/runbot/static/build/62744869-17-0/logs/build/html/applications/marketing/events.html)

StraubCreative commented on content/applications/marketing/events.rst:
```diff
@@ -7,9 +7,61 @@ Events
 .. seealso::
    - `Odoo Tutorials: Events <https://www.odoo.com/slides/surveys-63>`_
```
Will also need to edit the tags at the top, on line 1. 
@samueljlieber can advise on tech review.

StraubCreative reviewed (changes requested):
Hi @ksc-odoo 

Off to a good start. Please see the comments below and refer to the following directory parent docs for inspiration:
- [Removal strategies](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.html)
- [Subcontracting](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/manufacturing/subcontracting.html)
- [Getting started with Helpdesk](https://www.odoo.com/documentation/17.0/applications/services/helpdesk/overview/getting_started.html)

Ideally, these parent docs that house an entire application walk users through the basics of the UI, yes, and they also walk through the core features and configurations a user will need to know in order to get started. While walking through those basics, we can invite the user further into the directory, where the directory-level doc then acts as the "switchboard" for the app's repository of knowledge.

For all of those more detailed or advanced flows (such as creating your first event, booths, etc.) we can write a section in this doc  that contains a small blurb around the use case and why it's relevant to know, and add link(s) or a **seealso** that point to the larger doc which contains more information. [Here's a good example ](https://www.odoo.com/documentation/17.0/applications/finance/accounting.html) on what that looks like.

From here we have a couple options:
- simply fix the CRs below and then pass to tech review/merge (3pt)
- fix the CRs + expand the doc further (5pt), per the examples above, and re-request another review.

Thanks again and please let me know 🤙

ksc-odoo commented:
Thanks again for the always-helpful feedback and guidance with this PR, @StraubCreative -- I am formerly re-requesting another review from you, because I *believe* I've implemented the necessary elements to warrant another look. If there are further corrections I need to make, or if additional adjustments are required, just let me know, and I'll take care of 'em as soon as possible. Thanks! 🙏

StraubCreative commented on content/applications/marketing/events.rst:
```diff
@@ -4,12 +4,172 @@
 Events
 ======
 
+Generate and gather valuable data about events, and their attendees, which can then be used to
+improve decision-making and event-planning.
```
This sentence is not contextual to the doc.
Consider rewriting to provide an overview for what's specifically covered on this page.

StraubCreative commented on content/applications/marketing/events.rst:
```diff
@@ -4,12 +4,172 @@
+and other similar elements for events.
+
+.. note::
+   The :guilabel:`Jitsi Server Domain` field is populated, by default.
```
Can remove this **note** I think, unless we want to add more specific information about what the default server address is/means, how to change it, etc.

StraubCreative commented on content/applications/marketing/events.rst:
```diff
@@ -4,12 +4,172 @@
+Track and manage talks
+======================
+
+With Odoo *Events*, users can utilize a fully-integrated event website, where attendees can quickly
+access various tracks (talks, presentations, etc.), view entire agendas, and even propose talks for
+the event.
```
Parts of this are redundant or unnecessary. Trim to essential information.

StraubCreative reviewed (approved):
Hi @ksc-odoo 

Content looks ok. Approving with comments below for your consideration. 

Please work with @samueljlieber on the opening + closing tags of the RST file so the doc is both clickable and viewable in the side menu, and so the toctree links render nicely (screenshot below of current render, it's a bit messy).

Thanks!
![Screenshot 2024-05-28 at 6 38 01 PM](https://github.com/odoo/documentation/assets/5496809/ac22126b-dbc1-46ed-9346-a46b79dc20ef)

ksc-odoo commented:
Thanks for the helpful feedback, @StraubCreative -- I made those adjustments and believe this is now ready for Tech Review @samueljlieber (whenever you get a chance). Thanks! 👍

samueljlieber commented on content/applications/marketing/events/event_essentials.rst:
Since this file is removed, and the relevant content is now in `event.rst`, we need to add a redirect so that any users trying to access `events/event_essentials.rst` will be redirected to `events.rst` :)

Please add a line in `redirects/17.0.txt` in the marketing section
```
applications/marketing/events/event_essentials.rst applications/marketing/events.rst    # events/event_essentials --> events
```

samueljlieber reviewed (changes requested):
Hi @ksc-odoo! Nice job on this rewrite of the Event app overview. Your improvements look great to me! I caught a few small fixes and have some suggestions. I am requesting changes so I can double check the redirect. Please tag me for one more quick review once these have been addressed, thanks for your hard work– almost there!

ksc-odoo commented:
Hey @samueljlieber -- think this one is ready for another look from you, whenever you get a chance. Thanks!

samueljlieber reviewed (approved):
Thank you @ksc-odoo, your changes look good to me! Approving 👍
.....
@robodoo r+
